packages/drivers/routerlicious-driver/src/urlUtils.ts

```diff
@@ -17,3 +19,23 @@ export const parseFluidUrl = (fluidUrl: string): URLParse => {
  */
 export const replaceDocumentIdInPath = (urlPath: string, documentId: string): string =>
     urlPath.split("/").slice(0, -1).concat([documentId]).join("/");
+
+export const createFluidUrl = (domain: string, pathname: string): string =>
+    "fluid://".concat(domain).concat(pathname);
+
+export const replaceWithDiscoveryUrl = (resolvedUrl: IFluidResolvedUrl,
+    session: ISession,
+    parsedUrl: URLParse): void => {
+    if (session && session.ordererUrl.includes("https")) {
+        const replacementOrderUrl = new URL(session.ordererUrl);
```

Comment:
(nit) improper indentation?

Reply:
The indentation seems to be fine, could I know how to change it? Thanks.

Reply:
Sorry...I got it wrong :)